scripts/yum-wrapper

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+function hr() {
+  printf '%*s\n' "${COLUMNS:-$(tput cols)}" '' | tr ' ' -
+}
+
+function banner() {
+  hr
+  echo -e "\n  Installing additional packages is not supported. For more information, see:"
```

Comment:
Actually... it is referring to an action/verb and not the object. The action is "installing packages" which is a single thing.